extensions/smallrye-context-propagation/deployment/src/main/java/io/quarkus/smallrye/context/deployment/ManagedExecutorBuildItem.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkus.smallrye.context.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * This build item can be injected in an extension {@link io.quarkus.deployment.annotations.BuildStep BuildStep} to make sure
+ * the Quarkus {@link org.eclipse.microprofile.context.ManagedExecutor ManagedExecutor} instance is available for injection
+ * during bytecode recording. The executor can only be used from generated bytecode executed at
+ * {@link io.quarkus.deployment.annotations.ExecutionTime#RUNTIME_INIT ExecutionTime.RUNTIME_INIT}.
+ */
+public final class ManagedExecutorBuildItem extends SimpleBuildItem {
```

Comment:
Since it doesn't pass the `ManagedExecutor`, I would have named it `ManagedExecutorInitialized`.

Reply:
Done. That's a better name indeed.